armi/reactor/reactorParameters.py

```diff
@@ -723,42 +770,47 @@ def defineCoreParameters():
 
         pb.defParam(
             "breedingRatio",
-            units="N/A",
+            units=units.UNITLESS,
             description="Breeding ratio of the reactor",
             default=0.0,
         )
 
-        pb.defParam("ConvRatioCore", units="?", description="?")
+        pb.defParam(
+            "ConvRatioCore",
+            units=units.UNITLESS,
+            description="Conversion ratio of the core",
+        )
 
-        pb.defParam("absPerFisCore", units="?", description="?")
+        pb.defParam(
+            "absPerFisCore",
+            units=units.UNITLESS,
+            description="absorptions per fission in core",
+        )
 
         pb.defParam(
             "axialExpansionPercent",
-            units="%",
+            units=units.PERCENT,
             description="Percent of axial growth of fuel blocks",
             default=0.0,
         )
 
-        pb.defParam("coupledIteration", units="?", description="?", default=0)
-
-        pb.defParam("fisFrac", units="?", description="?")
-
-        pb.defParam("fisRateCore", units="?", description="?")
+        pb.defParam(
+            "coupledIteration",
```

Comment:
I'm not really sure what this parameter even is, but it seems like it belongs more on the reactor than the core, in line with thing like `timeNode`.

Not really related to your PR, just a thought.

Reply:
"coupledIteration" appears to be the major tool used in the past for our old "tight coupling", where you just define "number of iterations = 4" and move on with your life.

Tony has added more interesting "loose coupling" into ARMI, but for now `coupledIterations = 4` is still a thing people want to be able to do.  #shrug